back/internal/usersvc/internal/userdb/qryuser.go

```diff
@@ -32,13 +33,14 @@ func (s *UserDB) upsertUser(ctx cx, sid string, x *pb.AddUserReq, y *pb.UserResp
 		return fmt.Errorf("invalid uid")
 	}
 
-	// todo: be able to link roleIDs to users.
-	stmt, err := s.db.Prepare("INSERT INTO user (user_id, username, email, email_hold, altmail, altmail_hold, full_name, avatar, password) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?) ON DUPLICATE KEY UPDATE user_id = ?, username = ?, email = ?, email_hold = ?, altmail = ?, altmail_hold = ?, full_name = ?, avatar = ?, password = ?")
-	if err != nil {
-		return err
-	}
-
-	_, err = stmt.Exec(
+	qry := `
+		INSERT INTO user (user_id, username, email, email_hold, altmail, altmail_hold,
+			full_name, avatar, password) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?) 
+		ON DUPLICATE KEY UPDATE user_id = ?, username = ?, email = ?, email_hold = ?,
+			altmail = ?, altmail_hold = ?, full_name = ?, avatar = ?, password = ?
+	`
+	_, err := s.db.Exec(
+		qry,
```

Comment:
This long list of arguments is unwieldy. Are we using sqlx or something similar?